js_modules/dagster-ui/packages/ui-core/src/assets/auto-materialization/GlobalAutomaterializationContent.tsx

```diff
@@ -72,6 +73,11 @@ export const GlobalAutomaterializationContent = () => {
     [fetch, getVariables],
   );
 
+  // When the variables have changed (e.g. due to pagination), refresh.
```

Comment:
Hmm... I think I was relying on `useRefreshAtInterval` doing this... I think it should but also okay with requiring callsites to do it themselves. i'll leave it up to you.

Reply:
Ah, I had noticed that the `useRefreshAtInterval` is disabled (below) when a time range is applied, so I understood it to mean that the query only refreshes on the first page. (Which I think makes sense, since viewing an older page means the data should be static.)